lib/Objects/cpu_temperature_info.dart

```diff
@@ -0,0 +1,11 @@
+class CpuTemperatureInfo {
+  String temperature;
+  String label;
+
+  CpuTemperatureInfo({this.temperature, this.label});
+
+  CpuTemperatureInfo.fromJson(Map<String, dynamic> json) {
```

Comment:
If you make this static and return an object you can make all the variables in here final.  Makes the code safer.